solutions/coi-16-palinilap.mdx

```diff
@@ -0,0 +1,280 @@
+---
+id: coi-16-palinilap
+source: COI 2016
+title: Palinilap
+author: Kevin Sheng
+---
+
+[Official Analysis (C++)](https://github.com/mostafa-saad/MyCompetitiveProgramming/blob/master/Olympiad/COI/official/2016)
```

Comment:
link https://github.com/mostafa-saad/MyCompetitiveProgramming/blob/master/Olympiad/COI/official/2016/solutions.pdf as well?

Reply:
if you click my link solutions.pdf is right there?